corehq/apps/app_manager/management/commands/helpers.py

```diff
@@ -56,18 +60,34 @@ def add_arguments(self, parser):
             default=False,
             help="Perform the migration but don't save any changes",
         )
+        parser.add_argument(
+            '--restartable',
+            action='store_true',
+            default=False,
+            help="Uses two txt files to track progress of migrated docs, so that the command can be continue where"
+                 "it left off after being interrupted.",
+        )
 
     def handle(self, **options):
         start_time = time()
         self.options = options
-        if self.options['domain']:
+        can_continue_progress, domain_list_position, domains = self.try_to_continue_progress_if_restartable()
+        if can_continue_progress:
+            pass
+        elif self.options['domain']:
```

Comment:
it looks like the `domain` and `restartable` options are incompatible, which makes sense, but I'd add a guard or something to make sure we don't accidentally override the former